package.json

```diff
@@ -38,7 +38,8 @@
     "server": "http-server -o -c-1",
     "test": "nyc --reporter=lcovonly --reporter=text-summary mocha",
     "coverage": "if [ -z \"$COVERALLS_REPO_TOKEN\" ]; then cat coverage/lcov.info | coveralls; fi",
-    "release": "npm run build && semantic-release"
+    "release": "npm run build && semantic-release",
+    "eslint": "eslint **/*.js"
```

Comment:
you might be able to just call `eslint` without a glob since the config file now defines `files`

Reply:
Oh, I wasn't aware that this is a feature :D

Thanks